docs/resources/Channel.md

```diff
@@ -35,7 +35,7 @@ Represents a guild or DM channel within Discord.
 | default_auto_archive_duration? | integer                                                                    | default duration for newly created threads, in minutes, to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080                         |
 | permissions?                   | string                                                                     | computed permissions for the invoking user in the channel, including overwrites, only included when part of the `resolved` data received on a slash command interaction         |
 
-\* `rate_limit_per_user` also applies to thread creation. Users can send one message and create one thread during each `rate_limit_per_user` interval.
+\* `rate_limit_per_user` also applies to thread creation. Users can send one message and create one thread during each `rate_limit_per_user` interval. For thread channels, `rate_limit_per_user` is only returned if the field is set to a non-zero and non-null value. The abscene of this field in API calls and Gateway events should indicate that slowmode has been reset to the default value.
```

Comment:
If this note is for #3486 then isn't this just for creation and receiving the actual data? It was my understanding that the issue was regarding thread updates not sending the field if it had changed from non-default -> default value.

Reply:
Yeah, that was the intention, really all I did was change our clients to `channel.rateLimitPerUser = update.rateLimitPerUser ?? 0` which seemed pretty reasonable I guess?

Reply:
@Rapptz are you asking for the field to always be included in _dispatches_ like thread_update?